e2e/cephfs.go

```diff
@@ -331,6 +331,23 @@ var _ = Describe(cephfsType, func() {
 				})
 			}
 
+			By("verify PVC and App Binding with volumeBindingMode:WaitForFirstConsumer", func() {
+				err := createCephfsStorageClassWaitForFirstConsumer(f.ClientSet, f, true, nil)
+				if err != nil {
+					framework.Failf("failed to create CephFS storageclass: %v", err)
+				}
+
+				err = validatePVCAndAppWaitForFirstConsumer(pvcPath, appPath, f)
+				if err != nil {
+					framework.Failf("failed to validate CephFS pvc and application binding: %v", err)
+				}
+
+				err = deleteResource(cephFSExamplePath + "storageclass.yaml")
+				if err != nil {
+					framework.Failf("failed to delete CephFS storageclass: %v", err)
+				}
```

Comment:
please add a validation for omap and ceph resources like in other test to ensure that we are not leaving any stale resouces